### ISSUE TITLE
Label fails to underline keyboard shortcut key until Alt key is pressed

### ISSUE THREAD

**Issue by AArnott** [AvaloniaUI/Avalonia#9720]
**Describe the bug**

Windows used to always underline keyboard shortcut keys. Recent version have that behavior off by default, but it can be activated as described below. Most apps honor this setting, but Avalonia Labels do not.

**To Reproduce**

Configure Windows to always underline keyboard shortcut keys:
![image](https://user-images.githubusercontent.com/3548/208015829-6fd150eb-6099-4612-8dc4-9669c905acea.png)

Given a window with this xaml:
```xaml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="MyApp.MainWindow"
        Title="MyApp">
    <StackPanel>
		<Label Content="Select _textbox" Target="{Binding #txtbox}" />
		<TextBox Name="txtbox" />
	</StackPanel>
</Window>
```

Run the program

**Expected behavior**

The "t" in "Select **t**extbox" should be underlined immediately.

**Screenshots**

Without the alt key pressed:
![image](https://user-images.githubusercontent.com/3548/208015746-bd2bc414-938a-4d8d-bf74-90d92c780cf5.png)

With the alt key pressed:
![image](https://user-images.githubusercontent.com/3548/208015778-cae73f2c-9ffa-4c1e-b926-e76a7db8a8b8.png)

**Desktop (please complete the following information):**

- OS: Windows 11
- Avalonia version: 0.10.18

**Comment by grokys** [comment]
Yeah I'm afraid we don't yet have an API to surface that setting inside of Avalonia.

**Comment by maxkatz6** [comment]
At some point we might need to have some alternative of SystemParameters from WPF.
Unfortunately, most of the platforms don't have as many parameters available for apps, like Windows does.

**Comment by AArnott** [comment]
I wonder if some use of Microsoft.Extensions.Configuration might be useful to back the SystemParameters class. On Windows, it could include a data source that is the OS itself. For OSs that don't define as many of these parameters, end users or app developers could still control these settings by way of an appsettings.json or user settings.json file.
